l3kernel/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ this project uses date-based 'snapshot' version identifiers.
 
 ## [Unreleased]
 
+### Added
+- `\text_declare_<thing>case_mapping:nn(n)` to allow customisation
+  of case changing outcome
```

Comment:
as mentioned elsewhere, I suggest to always spell out the command names in full even if this means mentioning 3 or more names rather than some generic one ... people search for names and using grep etc and won't find them in this way.